package.json

```
@@ -1,8 +1,8 @@
 {
   "name": "vercel-storage",
   "version": "0.0.1",
-  "description": "The monorepo for everything Postgres at Vercel",
-  "homepage": "https://vercel.com",
+  "description": "The monorepo for everything Storage at Vercel",
+  "homepage": "https://vercel.com/storage",
```

```suggestion
  "homepage": "https://vercel.com",
```

The description should definitely be updated but the homepage is correct! If you look a couple of lines down, the repository is set to `github.com/vercel/storage`.